src/index.html

```diff
@@ -22,6 +22,375 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+        class="image"
+        src="././images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="navigation">
+        <ul class="list">
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="list__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="list__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__photo"
+          src="./images/imac.jpeg"
+          alt="IMac"
+        />
+
+        <p class="card__description">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+
+        <p class="card__number">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-amount">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```